compiler/docgen.nim

```diff
@@ -777,6 +777,28 @@ type DocFlags = enum
   kDefault
   kForceExport
 
+proc genSeeSrcRope(d: PDoc, path: string, line: int): Rope=
+  var seeSrcRope: Rope = nil
+  let docItemSeeSrc = getConfigVar(d.conf, "doc.item.seesrc")
+  if docItemSeeSrc.len > 0:
+    let path = relativeTo(AbsoluteFile path, AbsoluteDir getCurrentDir(), '/')
+    when false:
```

Comment:
Can the `when false:` block be removed ?. :eyes:

Reply:
pre-existing from this PR (was just moved); can be removed in future PR, maybe